setup.md

```diff
@@ -4,23 +4,22 @@ title: Setup
 ---
 
 > ## Data
-> Data for this lesson is from the Portal Project Teaching Database -
-> [available on FigShare](https://figshare.com/articles/Portal_Project_Teaching_Database/1314459).
->
-> We will use the six files listed below for the data in this lesson.
-> Download these files to your computer either by clicking
-> [this link](https://github.com/weecology/portal-teachingdb/archive/master.zip),
-> which will give you everything in a single compressed file.
-> You'll need to unzip this file after downloading it.
->
-> Or download each file individually with the following links:
->
+> Data for this lesson is from the Portal Project Teaching Database
+> [which can be found on FigShare](https://figshare.com/articles/Portal_Project_Teaching_Database/1314459).
```

Comment:
What do you think about moving link from "which can be found on FigShare" to "Portal Project Teaching Database"? Shall we, perhaps, delete "which can be found on FigShare" altogether?